extensions/amp-list/0.1/amp-list.js

```diff
@@ -65,11 +64,7 @@ export class AmpList extends AMP.BaseElement {
           if (!opts.credentials) {
             opts.requireAmpResponseSourceOrigin = false;
           }
-          const fetchPromise = batchedXhrFor(this.win).fetchJson(src, opts);
-          const fragment = getFragment(src).slice(1);
-          return fragment ?
-              fetchPromise.then(json => getPath(json, fragment)) :
-              fetchPromise;
```

Comment:
I noticed the `items` attribute performs mostly the same function as the fragment. Having both the fragment and the items attribute seemed a bit redundant too. I can put it back in if someone believes there's a use-case for both.